ci/verify_examples.sh

```diff
@@ -18,6 +18,8 @@ FLAKY_SANDBOXES=(
     # https://github.com/envoyproxy/envoy/issues/31333
     locality-load-balancing
     # https://github.com/envoyproxy/envoy/issues/28541
+    lua-cluster-specifier
+    # https://github.com/envoyproxy/envoy/issues/33533
```

Comment:
shouldn't the comment appear before the sandbox name?

Reply:
good catch - ill update